tools/ci_build/build.py

```diff
@@ -1155,6 +1156,9 @@ def generate_build_tree(
         cmake_args.append("-Donnxruntime_DNNL_ACL_ROOT=" + args.dnnl_acl_root)
     if args.build_wasm:
```

Comment:
add a check - if `enable_wasm_simd` is `False` but `enable_wasm_relaxed_simd` is `True`, raise an error.

Reply:
My original design is to allow `enable_wasm_simd` to be any value when `enable_wasm_relaxed_simd` is `True`. But I can add an error here to make it more clear as you said.